src/tr1d1um/http.go

```diff
@@ -52,8 +51,30 @@ func (sh ConversionHandler) ServeHTTP(response http.ResponseWriter, request *htt
 	}
 }
 */
+func ConversionHandler(resp http.ResponseWriter, req *http.Request){
```

Comment:
I added this just to double check I am on the right track with respect to the process of getting  the data from the request to the WDMP format to finally WRP.